### PR TITLE
Updated launch script to get bootnode id

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: scripts/network-launch/launch.sh [hunk 1]
```diff
@@ -82,10 +82,10 @@ start_node () {
   nodeName=${2}
   serviceName=${3}
   containerName=${4}
-  bootNodeID=$(curl -H 'Content-Type: application/json' --data '{ "jsonrpc":"2.0", "method":"system_localPeerId", "id":1 }' ${bootNodeIP}:9933 -s | jq '.result')
+  bootNodeID=$(curl -H 'Content-Type: application/json' --data '{ "jsonrpc":"2.0", "method":"system_localPeerId", "id":1 }' ${bootNodeHost}:9934 -s | jq '.result')
```
Comment (AndreiNavoichyk) [inline scripts/network-launch/launch.sh:85]: Add protocol `https://`

File: scripts/network-launch/launch.sh [hunk 2]
```diff
@@ -82,10 +82,10 @@ start_node () {
...
   while [ -z $bootNodeID ]; do
       echo "*** bootNodeID is empty "
-      bootNodeID=$(curl -H 'Content-Type: application/json' --data '{ "jsonrpc":"2.0", "method":"system_localPeerId", "id":1 }' ${bootNodeIP}:9933 -s | jq '.result')
+      bootNodeID=$(curl -H 'Content-Type: application/json' --data '{ "jsonrpc":"2.0", "method":"system_localPeerId", "id":1 }' ${bootNodeHost}:9934 -s | jq '.result')
```
Comment (AndreiNavoichyk) [inline scripts/network-launch/launch.sh:88]: Add protocol `https://`

File: scripts/network-launch/launch.sh (hunk 1 above)
Comment (karthikeyan-cere) [reply scripts/network-launch/launch.sh]: Added

File: scripts/network-launch/launch.sh (hunk 2 above)
Comment (karthikeyan-cere) [reply scripts/network-launch/launch.sh]: Added

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**